src/app/search/search-workflow-table/search-workflow-table.component.html

```diff
@@ -18,13 +18,20 @@
                 (workflow?.source.workflowName ? '/' + workflow?.source.workflowName : '')
             }}</a
           >
-          <div
-            class="truncate-text-2 size-small"
-            *ngIf="workflow.source.topicAutomatic"
-            [matTooltip]="workflow.source.topicAutomatic"
-            matTooltipPosition="left"
-          >
-            {{ workflow.source.topicAutomatic }}
+          <div class="truncate-text-2 size-small" [matTooltip]="workflow?.source.topicAutomatic" matTooltipPosition="left">
+            {{ workflow?.source.topicAutomatic }}
+            <a
+              *ngIf="workflow?.source.topicSelection === TopicSelectionEnum.AI"
+              [href]="Dockstore.DOCUMENTATION_URL + '/faq.html'"
+              target="_blank"
+              rel="noopener noreferrer"
+              data-cy="ai-bubble"
+              class="bubble ai-bubble"
+              matTooltip="This topic sentence was AI generated. Non-published workflows will not be sent for AI topic sentences."
```

Comment:
I find this confusing. It says the topic sentence was AI generated, but then says isn't an AI topic sentence, if it's not published.